app/views/BookDetail/index.tsx

```diff
@@ -136,12 +333,87 @@ function BookDetail() {
                                     />
                                 )}
                             </div>
+                            <div className={styles.actions}>
+                                {isAlreadyInCart && (
+                                    <div className={styles.cartDetails}>
+                                        <div className={styles.cartQuantityInfo}>
+                                            {`${bookDetail.book.cartDetails?.quantity} item(s) in the cart`}
+                                        </div>
+                                        <Button
+                                            className={styles.removeButton}
+                                            name="remove-from-cart"
+                                            variant="transparent"
+                                            icons={<IoTrash />}
+                                            onClick={handleRemoveFromCart}
+                                        >
+                                            Remove from Cart
+                                        </Button>
+                                    </div>
+                                )}
+                                <div className={styles.primaryAction}>
+                                    <NumberInput
+                                        className={styles.quantityInput}
+                                        label="Quanitity"
+                                        name="number-of-items"
```

Comment:
```suggestion
                                        name="numberOfItems"
```
If the name is not used set it as undefined